tests/run-make/external-crate-panic-handle-no-lint/rmake.rs

```diff
@@ -0,0 +1,16 @@
+// Defining a crate that provides panic handling as an external crate
+// could uselessly trigger the "unused external crate" lint. This test
```

Comment:
Is this lint *actually* even being checked in the original test (and thus this test)? There's no deny for this lint, either in source or as a `-D unused_extern_crate` cli flag, so I don't think it's actually being checked is it?

Reply:
You're right, similarly to my last incorrect comment, it's not checking that the patch _fixed something_, but rather that it did not _break something else_. It's important to remember.